src/lib/packager/clients/pkg_finish.rb

```diff
@@ -284,15 +285,24 @@ def sync_target_sources
       Pkg.TargetInitialize(Installation.destdir)
     end
 
-    # Set libzypp configuration to install the minimal amount of packages
+    # Set package manager configuration to install the minimal amount of packages
     #
     # @see Yast::Packager::CFA::ZyppConf#set_minimalistic!
-    def set_minimalistic_libzypp_conf
-      log.info("Setting libzypp configuration as minimalistic")
-      config = Packager::CFA::ZyppConf.new
-      config.load
-      config.set_minimalistic!
-      config.save
+    def set_minimalistic_pkg_conf
+      ["dnf", "libzypp"].each do |pkg|
+        file = "/etc/dnf/dnf.conf" if pkg == "dnf"
```

Comment:
Use the constants here (`Packager::CFA::DnfConf::PATH`) the path should be ideally defined in a single place to minimize possible changes.

Reply:
I would go even one step further and define constant mapping
```
PKG_MAPPING = {
  "dnf" => "/etc/dnf/dnf.conf",
  "libzypp" => "/etc/zypp/zypp.conf"
}
```

and then PKG_MAPPING.keys.each do |pkg| and so on.